server/main.lua

```diff
@@ -93,9 +93,9 @@ local function GetRaceId(name)
 end
 
 local function GenerateRaceId()
-    local RaceId = "LR-" .. math.random(1111, 9999)
+    local RaceId = "LR-" .. math.random(0000000, 9999999)
```

Comment:
```suggestion
    local RaceId = "LR-" .. math.random(0, 9999999)
```
`0000000` is equivalent to `0` (or change it to `1111111` idk what's better).